worldlimiter.lua

```diff
@@ -1,29 +1,50 @@
+local WorldLimiter_ScheduleInProgress = 0
```

Comment:
Right. Forgive me, these are among my first lines of Lua. I'll fix this in a moment.